src/AltinnCore/Designer/TypedHttpClients/AzureDevOps/Models/Build.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps.Enums;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace AltinnCore.Designer.TypedHttpClients.AzureDevOps.Models
+{
+    /// <summary>
+    /// Azure DevOps Build
+    /// </summary>
+    public class Build
+    {
+        /// <summary>
+        /// Build id
+        /// </summary>
+        [JsonProperty("id")]
+        public int Id { get; set; }
+
+        /// <summary>
+        /// The time that the build was started.
+        /// </summary>
+        [JsonProperty("startTime")]
+        public DateTime? StartTime { get; set; }
```

Comment:
It has been decided to not use start{Time/DateTime} at the end of parameters that hold datetime properties. Could this be changed to just Start and Finish ?

Reply:
This was unknown to me, good to know for future reference. However this model is used for communication with Azure DevOps so the model reflects what they respond with.